modules/cram/templates/deepvariant_call.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+create_bed () {
+  echo -e "!{bedContent}" > "!{bed}"
+}
+
+call_small_variants () {
+    local args=()
+    args+=("--model_type" "!{model}")
+    args+=("--ref" "!{reference}")
+    args+=("--reads" "!{cram}")
+    args+=("--output_vcf" "!{vcfOut}")
+    args+=("--num_shards" "!{task.cpus}")
+    args+=("--regions" "!{bed}")
+    args+=("--intermediate_results_dir" ".")
+    #args+=("--sample_name" "!{sampleName}")
```

Comment:
sampleName is declared in deepvariant.nf line 15. should it be enabled/disabled here?